test/test_jit.py

```diff
@@ -3481,7 +3480,7 @@ def func(a):
     def test_loop_unrolling(self):
         def fn(x):
             y = FIXME_zerol()
-            for i in range(x):
+            for i in range(int(x)):
```

Comment:
We can add back tensor-to-scalar implicit conversion,  but it will require emitBuiltin to distinguish between perfect matches and matches requiring casting, which is big enough that I don't want to pile on here. I've added it to follow up tasks.